docs/app/Examples/modules/Dropdown/Content/DropdownExampleDescription.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Dropdown } from 'semantic-ui-react'
+
+const DropdownExampleDescription = () => (
+  <Dropdown text='Filter' floating labeled button className='icon'>
+    {/* <i class="filter icon"></i> */}
```

Comment:
In the future I think a TODO would be appropriate. This way we have some tooling ability to backtrack and complete them.  Going to merge this as is for now, we can fix later!